### PR TITLE
Pract 22.adding landing page

### PR DESCRIPTION
**Pull request by Daria1911**

### CODE REVIEW

File: src/App.js
Comment (Alzhan-B) [inline src/App.js:1]: Hey Dasha! Everything looks good. One small question - on line 5, our port for front end will be 8000? Or you are listening the backend here?

**Review by Alzhan-B — APPROVED** [review]

File: src/App.js
Comment (Alejandro-Patino-Camargo) [reply src/App.js:1]: Here we are listening to the backend routes.

**Review by Alejandro-Patino-Camargo — APPROVED** [review]
looks good!

File: src/components/LandingPage.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-content: baseline;
+    align-items: center;
+}
+
+.navigation ul, .logo{
+    display: flex;
+    flex-direction: row;
```
Comment (nikkigraybeal) [inline src/components/LandingPage.css:12]: I think row is the default flex-direction value, so you can leave it off if you want. Totally ok to include it though!

File: src/components/LandingPage.js
Comment (nikkigraybeal) [inline src/components/LandingPage.js:1]: I think LandingPage.js and LandingPage.css can be renamed to Home.js and Home.css and moved in to a new folder called home. Then move this home folder into a new folder in /src called pages.

**Review by nikkigraybeal — COMMENTED** [review]
Great work Dasha! Just one issue with the file structure, but we can easily change that in the next PR that gets merged, I think!

**Pull request merged**